Doc/library/uuid.rst

```diff
@@ -95,25 +95,25 @@ which relays any information about the UUID's safety, using this enumeration:
    A tuple of the six integer fields of the UUID, which are also available as six
    individual attributes and two derived attributes:
 
-   +------------------------------+-------------------------------+
-   | Field                        | Meaning                       |
-   +==============================+===============================+
-   | :attr:`time_low`             | the first 32 bits of the UUID |
-   +------------------------------+-------------------------------+
-   | :attr:`time_mid`             | the next 16 bits of the UUID  |
-   +------------------------------+-------------------------------+
-   | :attr:`time_hi_version`      | the next 16 bits of the UUID  |
-   +------------------------------+-------------------------------+
-   | :attr:`clock_seq_hi_variant` | the next 8 bits of the UUID   |
-   +------------------------------+-------------------------------+
-   | :attr:`clock_seq_low`        | the next 8 bits of the UUID   |
-   +------------------------------+-------------------------------+
-   | :attr:`node`                 | the last 48 bits of the UUID  |
-   +------------------------------+-------------------------------+
-   | :attr:`time`                 | the 60-bit timestamp          |
-   +------------------------------+-------------------------------+
-   | :attr:`clock_seq`            | the 14-bit sequence number    |
-   +------------------------------+-------------------------------+
+   +-------------------------------+-------------------------------+
+   | Field                         | Meaning                       |
+   +===============================+===============================+
+   | :attr:`!time_low`             | the first 32 bits of the UUID |
```

Comment:
I'd prefer to use one of the techniques we used in https://github.com/python/cpython/commit/29fa7afef94d74e18d97485c085d1ccf80c16ca3 here, so that we add anchors to all these attribute definitions. Don't have a strong opinion as to whether you turn it into an attribute-definition list like we did with `sys._emscripten_info`, or keep it as a table like we did with `sys.flags`